skuba-update/skuba_update/skuba_update.py

```diff
@@ -116,7 +119,8 @@ def update():
 
     code = run_zypper_patch()
     if is_restart_needed(code):
```

Comment:
Unrelated to this PR but I'd probably name this method differently. It confused me quite a lot when I first looked at it as didn't remember the distinction we did with restart and reboot :P Probably something like `is_zypper_restart_needed` would have been way explicit :)